.github/ISSUE_TEMPLATE/feature-request.md

```diff
@@ -1,13 +1,16 @@
 ---
 name: Feature Request
 about: Suggest an idea/feature/enhancement
-title: ''
+title: '[Enhancement] A brief description of what you think should be added to BuildingGadgets'
```

Comment:
```suggestion
title: ''
```
Again, the only thing of value this adds is an encouragement to use [x] psudo-tags.